src/main/java/org/testmonkeys/jentitytest/matchers/EntityMatcher.java

```diff
@@ -48,4 +50,15 @@ public void describeTo(Description description) {
         description.appendText("Entities have same values in properties");
     }
 
+    private StringBuilder showControlChars(Object inputObj) {
```

Comment:
Move method to a utils class(as an option) - could take as parameter a map of string and it's replacement so it's generic, and could be used in StringComparator for removing symbols.